### PR TITLE
ML can now fire her melting blob point blanc

### PR DESCRIPTION
**Pull request by UnokiAs**
## About The Pull Request
ML now can fire point blanc.
Melting blob projectile now convert on impact with a dead human.
Some walk speed tweak
Remove the mobs pick target (It made her AI very sluggish...)
Reduce the spread a little and buff Adoration damage to not be like der freis with just dot

https://user-images.githubusercontent.com/82665345/185787727-69a84296-a6c1-409e-8c32-193eb66e5b9e.mp4

## Why It's Good For The Game
She can now use her projectile more.

## Changelog
:cl: Unoki
tweak: Melting speed, Melting Blob convert on hit, Melting fire point blanc, Adoration buff

/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
I don't think it's a good idea, it will only make fighting ML easier, since that's how retreat works, it will prevent them from attacking entirely, so you could just block their movement at a corner and get in free hits.

**Comment by UnokiAs** [comment]
> I don't think it's a good idea, it will only make fighting ML easier, since that's how retreat works, it will prevent them from attacking entirely, so you could just block their movement at a corner and get in free hits.

Alright I made her fire point blanc now

File: code/modules/projectiles/projectile/ego_bullets/aleph.dm [hunk 1]
```diff
@@ -9,10 +9,10 @@
 	name = "slime projectile"
 	icon_state = "slime"
 	desc = "A glob of infectious slime. It's going for your heart."
-	damage = 88
+	damage = 100
```
Comment (EgorDinamit) [inline code/modules/projectiles/projectile/ego_bullets/aleph.dm:12]: See below. It does damage over time, so with your buff it will be ~150 black damage.

Comment (EgorDinamit) [reply code/modules/projectiles/projectile/ego_bullets/aleph.dm:12]: I'm not saying it should be nerfed, but I'm not sure if that was your intent here.

Comment (UnokiAs) [reply code/modules/projectiles/projectile/ego_bullets/aleph.dm:12]: Eh I saw the impact damage of ego_magicbullet (80 Black) while having a better firerate/projectile speed but yeah I can revert the damage buff if needed

Comment (EgorDinamit) [reply code/modules/projectiles/projectile/ego_bullets/aleph.dm]: Wasn't needed, but I'll see how the weapon performs in the game.

Comment (UnokiAs) [reply code/modules/projectiles/projectile/ego_bullets/aleph.dm]: I revert it, I will leave it to someone that eventually do a ego weapon rebalance better have one big pr like the armor one

**Pull request merged**